Resources/public/js/views/ez-confirmboxview.js

```diff
@@ -163,6 +165,17 @@ YUI.add('ez-confirmboxview', function (Y) {
             },
 
             /**
+             * Details of the confirmBox
```

Comment:
you should document that this attribute is supposed to contain an html fragment that won't be escaped while rendering the confirm box view.